### PR TITLE
NTP-1114: Team review mop up - layout and styling issues

### PR DESCRIPTION
**Pull request by kieranohanlon**
## Link to Jira ticket

https://dfedigital.atlassian.net/browse/NTP-1114

## Things to check

- [ ] Title is in the format "NTP-XXX: Short description" where NTP-XXX is the Jira ticket reference
- [ ] Code and tests follow the [coding standards](/docs/coding-standards.md)
- [ ] `dotnet format` has been run in the repository root
- [ ] Test coverage of new code is at least 80%
- [ ] All UI related acceptance criteria specified in the ticket have been added to the relevant [cypress test feature file(s)](/UI/cypress/e2e/)
- [ ] Unless explicitly mentioned in the ticket, all UI work should have been tested as working on desktop and mobile resolutions with JavaScript on and off
- [ ] Database migrations can be applied to the current codebase in main without causing exceptions
- [ ] Debug logging has been added after all logic decision points
- [ ] All [automated testing](/README.md#testing) including accessibility and security has been run against your code
- [ ] **PR deployment has been signed off by wider team**

### CODE REVIEW

File: UI/Pages/Enquiry/Respond/EnquirerResponse.cshtml [hunk 1]
```diff
@@ -1,125 +1,122 @@
...
+    <govuk-error-summary>
+      <govuk-error-summary-item asp-for="ErrorMessage" />
+    </govuk-error-summary>
+
+    <h1 class="govuk-heading-l">
+      View @Model.Data.TuitionPartnerName’s response
+    </h1>
+
+    <p class="govuk-body app-print-hide">
+      You can view and compare their response to your tuition requirements.<br>
```
Comment (hasan3d) [inline UI/Pages/Enquiry/Respond/EnquirerResponse.cshtml:26]: It should be `<br/>`

File: UI/Pages/Enquiry/Respond/Response.cshtml [hunk 2]
```diff
@@ -2,204 +2,187 @@
...
-    <input asp-for="Data.SupportReferenceNumber" type="hidden"/>
-    <input asp-for="Data.TuitionPartnerName" type="hidden"/>
-    @if (Model.Data.EnquiryKeyStageSubjects.Any())
-    {
-        @for (int i = 0; i < Model.Data.EnquiryKeyStageSubjects.Count(); i++)
+
+        <hr class="govuk-section-break govuk-section-break--visible govuk-!-padding-bottom-0 govuk-!-margin-top-0 govuk-!-margin-bottom-7">
+
+        <h2 class="govuk-heading-s govuk-!-margin-bottom-0">
+          Tuition type:
```
Comment (hasan3d) [inline UI/Pages/Enquiry/Respond/Response.cshtml:86]: It doesn't match the `Tuition of type:` we display for the enquirer view response page. It should be either `Tuition type:` or `Tuition of type:` for both pages to be consistent.

Comment (kieranohanlon) [reply UI/Pages/Enquiry/Respond/Response.cshtml]: Agreed - I have updated all that I could find to be "Type of tuition"

**Review by hasan3d — APPROVED** [review]
lgtm

**Pull request merged**